javascript/javascript1/week1/week1.js

```diff
@@ -6,63 +6,101 @@ const age = yearFuture - yearOfBirth;
 console.log("You will be " + age + " years old in " + yearFuture);
 
 //Goodboy-Oldboy (A dog age calculator)
-console.log("******************* Goodboy-Oldboy (A dog age calculator) **********");
+console.log(
+  "******************* Goodboy-Oldboy (A dog age calculator) **********"
+);
 const dogYearOfBirth = 2017;
 const dogYearFuture = 2027;
 let dogYear;
+let yearType;
 const shouldShowResultInDogYears = false;
-if(shouldShowResultInDogYears){
-    dogYear = (dogYearFuture - dogYearOfBirth) * 7;
-    console.log("Your dog will be "+ dogYear + " dog years old in " +dogYearFuture);
+if (shouldShowResultInDogYears) {
+  dogYear = (dogYearFuture - dogYearOfBirth) * 7;
+} else {
+  dogYear = dogYearFuture - dogYearOfBirth;
 }
-else{
-    dogYear = (dogYearFuture - dogYearOfBirth);
-    console.log("Your dog will be "+ dogYear + " human years old in " +dogYearFuture);
+if (shouldShowResultInDogYears) {
+  yearType = "dog";
+} else {
+  yearType = "human";
 }
-
+console.log(
+  "Your dog will be " +
+    dogYear +
+    " " +
+    yearType +
+    " years old in " +
+    dogYearFuture
+);
```

Comment:
Sorry, I've confused you with my previous comments in relation to what is template and string literal. What you have used here is a string literal - it's useful for short things, but once you get more than 1 variables it can start looks messy. That's where template literals can serve you well. In this case:
```suggestion
console.log(`Your dog will be ${dogYear} ${yearType} years old in ${dogYearFuture}`);
```